### DISCUSSION TITLE
Optional core-js polyfill and change javascript utility from lodash to just

### DISCUSSION THREAD

**Discussion by sadeghbarati** [idiotWu/smooth-scrollbar#406] [Ideas]
https://bundlephobia.com/package/smooth-scrollbar@8.7.0

**optional** `core-js` for the next release will be  the best option because ppl may already have core-js polyfills on their project (duplicate polyfills code)
 or even don't wanted to polyfill _older browsers_
 
 ---
 
 optional: use **just** utility instead of **lodash**

https://github.com/angus-c/just
https://github.com/puruvj/all-of-just
lodash vs just: https://github.com/angus-c/just/issues/4

**Comment by dolphin-wood** [comment]
We are going to remove core-js in the next release 😆
